src/tests/banners/ContributionsBannerTests.ts

```diff
@@ -0,0 +1,55 @@
+import fetch from 'node-fetch';
+import {
+    BannerTestGenerator,
+    BannerTest,
+    BannerTargeting,
+    BannerPageTracking,
+    RawTestParams,
+    RawVariantParams,
+    BannerVariant,
+    // BannerContent,
+} from '../../types/BannerTypes';
+
+export const ContributionsBannerPath = 'contributions-banner.js';
+
+const ContributionsBannerContentUrl =
+    'https://gu-contributions-public.s3-eu-west-1.amazonaws.com/banner/CODE/banner-tests.json';
+
+const ContributionsBannerTest = (testParams: RawTestParams): BannerTest => {
+    return {
+        name: testParams.name,
+        bannerType: 'contributions',
+        testAudience: testParams.userCohort,
```

Comment:
Unfortunately the types do not match here.
The `UserCohort` type is here: https://github.com/guardian/support-admin-console/blob/07c47b4210d434d13e04832a041c73d9d44b1023/app/models/UserCohort.scala

Let's change the definition of `BannerAudience` in this repo now to match the `UserCohort` type

Reply:
✅